src/components/booklist.js

```diff
@@ -1,29 +1,15 @@
+import { useSelector } from 'react-redux';
 import CreateBook from './createbook';
 import Book from './displaybook';
 
 export default function BookList() {
-  const bookList = [
-    {
-      Id: 1,
-      bookname: 'React Book',
-      auther: 'shah zaman',
-    },
-    {
-      Id: 2,
-      bookname: 'Next js Book',
-      auther: 'shah zaman',
-    },
-    {
-      Id: 3,
-      bookname: 'Angular Book',
-      auther: 'shah zaman',
-    },
-  ];
+  const books = useSelector((state) => state.books);
+  console.log(books);
   return (
     <>
       {
-        bookList.map((book) => (
-          <div key={book.Id}>
+        books.map((book) => (
```

Comment:
- [ ] Good work fixing the error in the browser! But yes, sometimes happens that you fix one error and there's a new one. That's progress! 
Kindly fix the new error in your browser console to make your console really clean 💯 

![image](https://github.com/sheezu900/Bookstore/assets/93630700/f72efed6-a028-476b-8cb2-c61c36f24809)